oneanddone/functional_tests/conftest.py

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+import requests
+
+
+@pytest.fixture(scope='session')
+def capabilities(capabilities):
+    capabilities.setdefault('tags', []).append('oneanddone')
+    return capabilities
+
+
+@pytest.fixture
+def persona_test_user():
+    return requests.get('http://personatestuser.org/email').json()
+
+
+@pytest.fixture(scope='function')
+def new_user(persona_test_user):
+    return {
+        'email': persona_test_user['email'],
+        'password': persona_test_user['pass'],
+        'name': persona_test_user['email'].split('@')[0],
+        'username': persona_test_user['email'].split('@')[0],
+        'url': 'http://www.mozilla.org/'
+    }
+
+
+@pytest.fixture(scope='session')
+def base_url(base_url, live_server):
+    if base_url is None:
+        return live_server.url
+    return base_url
```

Comment:
I know we already amused ourselves with the various ways to write this, but here's another that you might actually like:

```python
def base_url(base_url, live_server):
    return base_url or live_server.url
```

Also, did you try without the `scope='session'`?